Automattic/pocket-casts-android
# Display sharing feedback in a snackbar

## Description

This PR displays sharing feedback messages in a Snackbar instead of Toast. I didn't use the default Snackbar implementation because I have to control theming based on a podcast and the deafult implementation doesn't have an easy way to control text's color.

Designs: HR2vcQrWcS0scsolKZBitg-fi-2345_20311

## Testing Instructions

1. Share a podcast to Instagram.
2. You should see a snackbar feedback message.
3. Share an episode to Instagram.
4. You should see a snackbar feedback message.
5. Share an episode position to Instagram.
6. You should see a snackbar feedback message.

## Screenshots or Screencast 

![share_2936570251206927203](https://github.com/user-attachments/assets/373a0ff6-9d72-47c9-b3bd-6410f365f406)

## Checklist
 
#### I have tested any UI changes...
